en/core-libraries/email.rst

```diff
@@ -28,8 +28,16 @@ After you've loaded ``Email``, you can send an email with the following::
         ->subject('About')
         ->send('My message');
 
-Since ``Email``'s setter methods return the instance of the class, you are able
-to set its properties with method chaining.
+Since ``Email``'s setter methods return the instance of the class, you are able to set its properties with method chaining.
+
+``Email`` has several methods for defining recepients - ``to()``,``cc()``,``bcc()``, ``addTo()``, ``addCc()`` and ``addBcc()``.  The main difference being that the first three will overwrite what was already set and the later will just add more recepients to their respective field.
```

Comment:
@markstory can you take care of it and I will know how next time? Or point me to the docs where I can read about it and I'll do it.